perception/image_projection_based_fusion/src/roi_cluster_fusion/node.cpp

```diff
@@ -190,7 +193,34 @@ void RoiClusterFusionNode::fuseOnSingleImage(
         autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN) {
       output_cluster_msg.feature_objects.at(index).object.classification =
         feature_obj.object.classification;
+
+      // Update existence_probability for fused objects
+      if (
+        output_cluster_msg.feature_objects.at(index).object.existence_probability <
+        min_roi_existence_prob_) {
+        output_cluster_msg.feature_objects.at(index).object.existence_probability =
+          min_roi_existence_prob_;
+      }
+    }
+
+    // fuse with unknown roi
+    if (
```

Comment:
[nits]
This is just a piece of coding advice.

To improve readability, when dealing with multiple 'if' statements with long conditions, it is recommended to define a boolean variable with a descriptive name representing the condition. Then, utilize this variable within the corresponding 'if' statements.
i.e. 
```
bool is_label_unknown = (output_cluster_msg.feature_objects.at(index).object.classification.front().label == autoware_auto_perception_msgs::msg::ObjectClassification::UNKNOWN);
if (is_label_unknown) {
  ...
```

Reply:
@miursh Thank you for your comments. I changed here https://github.com/autowarefoundation/autoware.universe/pull/4123/commits/4cf3efdf0d5a42dfd778bf9e2e50e2f4cdf53353